ios/uPortMobile/Classes/EthereumSigner.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2018 ConsenSys AG
+//
+// This file is part of uPort Mobile App.
+//
+// uPort Mobile App is free software: you can redistribute it and/or modify
```

Comment:
Would be nice to indent all lines below as the standard Xcode header, so adding one space before each line.